package.json

```diff
@@ -10,6 +10,10 @@
     "build": "webpack -p --progress --config webpack.config.prod.js"
   },
   "devDependencies": {
+    "babel-core": "^6.10.4",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-transform-runtime": "^6.8.0",
+    "babel-preset-es2015": "^6.9.0",
```

Comment:
I think the new recommended way of doing that is using `babel-env` (just a note for another PR)

Reply:
👍 just copied from `udata` for now.